tests/Makefile.am

```diff
@@ -363,6 +363,7 @@ TESTS +=  \
 	dircreate_off.sh \
 	imuxsock_legacy.sh \
 	imuxsock_logger.sh \
+	imuxsock_logger_ratelimit.sh \
```

Comment:
you need to add the new file to EXTRA_DIST as well (I know..., but that's historical and was not changed til now as it's quite some effort)